docs/community.rst

```diff
@@ -1,4 +1,35 @@
 .. _community:
 
 Community
-=========
\ No newline at end of file
+=========
+
+.. _code_of_conduct:
+
+Code of conduct
+---------------
+
+This project has adopted the `GitHub community guidelines 
+<https://help.github.com/en/github/site-policy/github-community-guidelines>`_.
+
+.. _communication:
+
+Real-time communication
+-----------------------
+
+For real time communication please join us on
+`Gitter <https://gitter.im/fairlearn/community>`_.
+All are welcome!
+
+.. _developer_calls:
+
+Developer Calls
+---------------
+
+We have a monthly contributor meeting every second Thursday of the month at
+`3pm UTC <http://time.unitarium.com/utc/3pm>`_. All are welcome! Please reach
+out via
+`Gitter <https://gitter.im/fairlearn/community>`_ and we will forward the
+invitation to you.
+We keep notes from each meeting in the 
+`fairlearn-proposals repository <https://github.com/fairlearn/fairlearn-proposals>`_
+in the :code:`developer_calls` directory.
```

Comment:
I was thinking about this. I'm okay storing it in `fairlearn-proposals` for now, but it doesn't quite fit the purpose of that repo. Two options:
* we could rename fairlearn-proposals
* or we could also save it in this repository, under `docs`. Then we could directly show this as part of our documentation--and index it under `docs/community`--perhaps hierarchically by year?

Reply:
Sure, we can have it in this repo.

Reply:
Added a directory/page for this. The content for the first call will go in in a separate PR.